docs/pom.xml

```diff
@@ -93,180 +82,52 @@
         </configuration>
       </plugin>
       <plugin>
-        <groupId>com.github.kongchen</groupId>
-        <artifactId>swagger-maven-plugin</artifactId>
-        <configuration>
-          <encoding>UTF-8</encoding>
-        </configuration>
+        <artifactId>maven-resources-plugin</artifactId>
         <executions>
           <execution>
-            <id>api-swagger-core</id>
-            <phase>prepare-package</phase>
-            <goals>
-              <goal>generate</goal>
-            </goals>
-            <configuration>
-              <apiSources>
-                <apiSource>
-                  <attachSwaggerArtifact>true</attachSwaggerArtifact>
-                  <outputFormats>json</outputFormats>
-                  <swaggerDirectory>${project.build.directory}/swagger</swaggerDirectory>
-                  <swaggerFileName>swagger-core</swaggerFileName>
-                  <locations>
-                    <location>io.atlasmap.service.AtlasService</location>
-                  </locations>
-                  <info>
-                    <title>AtlasMap Design Time Service :: Core</title>
-                    <version>${project.version}</version>
-                  </info>
-                </apiSource>
-              </apiSources>
-            </configuration>
-          </execution>
-          <execution>
-            <id>api-swagger-java</id>
-            <phase>prepare-package</phase>
+            <id>default-cli</id>
+            <phase>generate-resources</phase>
```

Comment:
Hi @zregvart, is this on purpose to merge all services into a single doc? wondering if there's an easy way to put sub-sections per service.

Reply:
Newer plugin doesn't look much different - I'll split the swagger doc like before.

Reply:
I think it was just simpler for me to do.